File: baekjoon/문자열/10809.js
const readline = require('readline')
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
})

let input = []

rl.on('line', (line) => {
  input.push(line)
}).on('close', () => {
  const S = input[0]
  const alphabets = [
    'a',
    'b',
    'c',
    'd',
    'e',
    'f',
    'g',
    'h',
    'i',
    'j',
    'k',
    'l',
    'm',
    'n',
    'o',
    'p',
    'q',
    'r',
    's',
    't',
    'u',
    'v',
    'w',
    'x',
    'y',
    'z',
  ]
  let result = ''
  for (i = 0; i < alphabets.length; i++) {
    const alphabet = alphabets[i]
    let location = null
    for (j = 0; j < S.length; j++) {
      if (alphabet === S[j]) {
        location = j
        break
      }
    }
    if (location === null) {
      result += `-1 `
    } else {
      result += `${location} `
    }
  }
  console.log(result)
  process.exit()
})

File: baekjoon/단계별로 풀어보기/문자열/10809.py
S = input()
alphabets = [
    'a',
    'b',
    'c',
    'd',
    'e',
    'f',
    'g',
    'h',
    'i',
    'j',
    'k',
    'l',
    'm',
    'n',
    'o',
    'p',
    'q',
    'r',
    's',
    't',
    'u',
    'v',
    'w',
    'x',
    'y',
    'z',
  ]
result = ''
for i in alphabets:
  num = ''
  for idx, j in enumerate(S):
    if i == j:
      num = idx
      break
  if num == '':
    result = result + '-1 '
  else:
    result = result + f'{num} '
print(result)